.github/ISSUE_TEMPLATE/FEATURE_REQUEST.yml

```
@@ -0,0 +1,45 @@
+name: ✨ Feature Request
+description: Request a new feature
+labels: ["enhancement", "needs-triage"]
+# assignees:
+#   - octocat
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Hi there 👋,
+
+        Thank you for requesting a new feature!
+  - type: checkboxes
+    id: type
+    attributes:
+      label: Feature Type
+      description: What do you want to request?
+      options:
+        - label: a new resource for an upstream API endpoint
+          required: false
+        - label: a new datasource for an upstream API endpoint
+          required: false
+        - label: align attributes with an upstream API endpoint
+          required: false
+        - label: something else
+          required: false
+  - type: textarea
+    id: description
+    attributes:
+      label: Feature Description
+      description: Please describe your feature ideas in as much detail as possible.
+    validations:
+      required: true
+  - type: textarea
+    id: example-configuration
+    attributes:
+      label: Example Terraform Configuration
+      description: Please provide an example configuration for how it could be implemented.
```

I think these can be combined. (This field is ok to make required obviously).

```suggestion
      description: Please describe your feature ideas in as much detail as possible, including an example configuration.
    validations:
      required: true
```